developer-docs/upgrading_kubernetes.md

```diff
@@ -1,10 +1,10 @@
 # Upgrade Kubernetes Process
 
-From time to time we need to update the version of Kubernetes used by RKE2. This document serves as a how-to for that process and constitutes a "release". The following steps are laid out in order.
+From time to time, we need to update the version of Kubernetes used by RKE2. This document serves as a how-to for said process and elaborates on what constitutes a "release". Although the steps described in the document are laid out in order, the overall flow looks this way:
 
-This process will need to be done any time a new release is needed.
+![Upgrading Kubernetes](upgrading_kubernetes.svg)
```

Comment:
I don't know if the top of the document is the best place for the image.

Reply:
I did it imitating [Kubernetes contributors documentation](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) layout, also thinking on newcomers that rather want to see an image before jumping on the docs. Either way, I can move it where you see it fit

Reply:
Let's move it down to the bottom but right above the "Release Process" section. I think where at the point now that a table of contents would be extremely helpful.

Reply:
I generated a TOC, please take a look